This gets rid of silly warnings about compiling without debugging symbols.

--- petsc-3.1.orig/config/PETSc/utilities/debugging.py	2006-03-20 16:51:11.000000000 -0500
+++ petsc-3.1/config/PETSc/utilities/debugging.py	2006-04-04 19:27:25.000000000 -0400
@@ -26,17 +26,7 @@
     # should do error checking
     if self.framework.argDB['with-errorchecking']:
       self.addDefine('USE_ERRORCHECKING',1)
-    else:
-      self.logPrintBox('     WARNING! Compiling PETSc with NO error checking/exception handling, \n \
-                    this should only be done for timing and production runs where you DO NOT \n \
-                    use PETSc exceptions. All development should be done when configured using \n \
-                    --with-errorchecking=1')          
-
     self.debugging = self.compilerFlags.debugging
-    if not self.debugging:
-      self.logPrintBox('          WARNING! Compiling PETSc with no debugging, this should \n \
-               only be done for timing and production runs. All development should \n \
-               be done when configured using --with-debugging=1')
 
   def configure(self):
     self.executeTest(self.configureDebugging)
